src/metabase/driver/util.clj

```diff
@@ -316,6 +316,22 @@
                  db-details
                  secret-names->props))))
 
+(def official-drivers
+  "List of all official drivers"
+  ["bigquery-cloud-sdk" "druid" "googleanalytics" "h2" "mongo" "mysql" "postgres" "presto" "presto-jdbc" "redshift" "snowflake" "sparksql" "sqlite" "sqlserver"])
```

Comment:
If you wanted to be extra clever you could could just read the list of directories inside `modules/drivers` and calculate this programmatically (you'd have to add H2/MySQL/Postgres to it still). If you mark it `:^const` it will be done at compile time and thus will already be pre-calculated in the JAR when we AOT compile.

It might be nice so this list can't get out of sync e.g. if we add a new driver in the future

Reply:
This is indeed fancy, appreciate the tip @camsaul !
I'll try to do this on a follow-up PR (need to learn and practice using `:^const` first).